streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -141,7 +141,7 @@
     private final QueryableStoreProvider queryableStoreProvider;
     private final Admin adminClient;
 
-    private GlobalStreamThread globalStreamThread;
+    protected GlobalStreamThread globalStreamThread;
```

Comment:
Similar to stream threads, making it accessible directly from KafkaStreamsTest.

Reply:
package-private should also work, right?